ThePensionsRegulator.Frontend/HtmlGeneration/ComponentGenerator.TprSearchInput.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace ThePensionsRegulator.Frontend.HtmlGeneration
+{
+    public partial class ComponentGenerator
+    {
+        internal const string DefaultSearchLabel = "Search Q&As";
+
+        public TagBuilder GenerateTprSearchInput(string? searchLabel = null)
+        {
+            if (string.IsNullOrWhiteSpace(searchLabel))
+            {
+                searchLabel = DefaultSearchLabel;
+            }
+
+            var outer = new TagBuilder("div");
+            outer.AddCssClass("govuk-grid-row");
+
+            var button = new TagBuilder("button");
+            button.AddCssClass("govuk-button");
+            button.AddCssClass("tpr-button--no-next-step");
+            button.Attributes.Add("id", "ask-button");
```

Comment:
The comments about hard-coded ids and non-localisable text apply here too.

If you're going to hard-code ids, consider prefixing with the component `tpr-search-results-` like you do with classes, so that other things could have an `ask-button` without conflicting.